apps/builder/app/builder/features/breakpoints/find-initial-width.ts

```diff
@@ -11,12 +12,24 @@ export const findInitialWidth = (
   selectedBreakpoint: Breakpoint,
   workspaceWidth: number
 ) => {
-  // When on a base breakpoint, we want to show maximum width canvas
-  if (
-    selectedBreakpoint.minWidth === undefined &&
-    selectedBreakpoint.maxWidth === undefined
-  ) {
-    return workspaceWidth;
+  if (isBaseBreakpoint(selectedBreakpoint)) {
```

Comment:
If the only breakpoint is the base breakpoint it should return workspaceWidth no returns 320